src/WebJobs.Extensions.CosmosDB/Config/ObjectCosmosSerializer.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+// Adapted from: https://github.com/Azure/azure-cosmos-dotnet-v3/blob/master/Microsoft.Azure.Cosmos/src/Serializer/CosmosJsonDotNetSerializer.cs
+using System;
+using System.IO;
+using Azure.Core.Serialization;
+using Microsoft.Azure.Cosmos;
+
+namespace Microsoft.Azure.WebJobs.Extensions.CosmosDB
+{
+    /// <summary>
+    /// This is a wrapper that allows us to use cosmosDB options ObjectSerializer to create a CosmosSerializer.
+    /// </summary>
+    internal sealed class ObjectCosmosSerializer : CosmosSerializer
```

Comment:
I think we may want to inherit `CosmosLinqSerializer`.

Reply:
`CosmosLinqSerializer` also inherits from `CosmosSerializer`. Is there any added benefit to using `CosmosLinqSerializer`?

For context, we have implemented same serializer on Worker side too - https://github.com/Azure/azure-functions-dotnet-worker/blob/main/extensions/Worker.Extensions.CosmosDB/src/WorkerCosmosSerializer.cs

Reply:
I think `CosmosLinqSerializer` is needed for the LINQ IQueryable support with Cosmos.

Reply:
The tests ran fine. This code will only execute if host.json has `dateParseHandling` serializerSettings.